instructure/canvas-ios
# Remove ExternalToolManager

refs: MBL-14562
affects: student, teacher
release note: none

Test plan:
- Launch an LTI tool from Course Nav
  - narmstrong > s > CS 1400 > Google Drive (it should ask for auth)
- Launch an LTI tool from a link in a page
  - narmstrong > s > CS 1400 > Pages > Links to Stuff > External Tool
